nested_sequence/text_classification/config.py

```diff
@@ -0,0 +1,39 @@
+__all__ = ["TrainerConfig", "ModelConfig"]
+
+
+class TrainerConfig(object):
+
+    # whether to use GPU for training
+    use_gpu = False
+    # the number of threads used in one machine
+    trainer_count = 1
+
+    # train batch size
+    batch_size = 32
+
+    # number of pass during training
+    num_passes = 10
+
+    # learning rate for optimizer
+    learning_rate = 1e-3
+
+    # learning rate for L2Regularization
```

Comment:
Please do not call it learning rate. Commonly this parameter is called the decay rate.